pysqldb3/sql.py

```diff
@@ -204,3 +204,44 @@
 comment on table {s}.{t} is '{t} created by {u} on {d}
 - imported using pysql module -'
 """
+
+GPKG_DEL_INDICES_QUERY_PG = r"""
+select t.relname as table_name, n.nspname as schema_name, i.relname as index_name, a.attname as column_name
+from pg_class t
+join pg_index ix
+on t.oid = ix.indrelid
+join pg_class i
+on i.oid = ix.indexrelid
+join pg_attribute a
+on a.attrelid = t.oid and a.attnum = ANY(ix.indkey) 
+join pg_catalog.pg_namespace n 
+on  t.relnamespace = n.oid 
+where
+    t.relkind = 'r'
+    and n.nspname != 'pg_catalog'
+    and t.relname = '{t}'
+    and n.nspname = '{s}'
+"""
+
+GPKG_DEL_INDICES_QUERY_MS = r"""
+    SELECT
+        '{t}' as table_name,
+        a.name AS index_name,
+        COL_NAME(b.object_id,b.column_id) AS Column_Name,
+        type_desc
+    FROM
+        sys.indexes AS a
+    INNER JOIN
+        sys.index_columns AS b
+        ON a.object_id = b.object_id AND a.index_id = b.index_id
+    WHERE
+        a.is_hypothetical = 0 
+        AND a.object_id = OBJECT_ID('{s}.{t}')
+        AND type_desc !='SPATIAL' -- taken care of by GDAL
+"""
```

Comment:
are these needed? any reason you cant use the `SHP_DEL_INDICES_QUERY_PG` / `SHP_DEL_INDICES_QUERY_MS`?